src/app/app.module.ts

```diff
@@ -179,6 +182,7 @@ export type StoreType = {
     ApiLocatorService,
     AreaService,
     authApiUrlProvider,
+    witApiProxyProvider,
```

Comment:
This list is alphabetical, please move it down.

Reply:
Done